src/Api/Models/Request/Organizations/OrganizationUpdateRequestModel.cs

```diff
@@ -12,6 +12,7 @@ public class OrganizationUpdateRequestModel
         public string Name { get; set; }
         [StringLength(50)]
         public string BusinessName { get; set; }
+        [Obsolete("2022-08-03 Moved to Org SSO request model, left for backwards compatability")]
```

Comment:
Do we have a tech debt task for this in the EC pod? It's good to include the jira issue tag here, and ensure the work gets schedule for the release after.

Reply:
We do now https://bitwarden.atlassian.net/browse/EC-489 and its referenced in the Obsolete message now. (Same for Client PR as well)